packages/ui/src/molecules/Incentives/IncentivesPure.tsx

```diff
@@ -0,0 +1,30 @@
+import type { HTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface IncentivesPureProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+}
+
+const IncentivesPure = forwardRef<HTMLDivElement, IncentivesPureProps>(
+  function IncentivesPure(
+    { testId = 'store-incentives', children, ...otherProps },
+    ref
+  ) {
+    return (
+      <section
```

Comment:
This looks good semantically, but I'm not sure we should hardcode a `section` element here... It should be up to the developer if they want to wrap it in a section. I was also reading about the `section` element and this article mentions they [should always have a heading (with a few exceptions)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#:~:text=Sections%20should%20always%20have%20a%20heading%2C%20with%20very%20few%20exceptions.), which isn't our case here... Not sure! 😵

Reply:
Good point @filipewl. After discussing with @igorbrasileiro we thought that using `section` would avoid the necessity of add a `role` for the wrapper at the same time it would bring some semantics, and it came after the a11y test failed because a `role` attribute was missing in the wrapper component (`IncentivesPure`). I'm not sure how we can solve this without using the `role` attribute neither the `section` tag...

Reply:
I see! Thanks for the explanation. I'm fine with leaving this as it is then!

Reply:
According to the child HTML elements (IncentivePures ul/li), this wrapper element should have an HTML tag with a proper role. So, we did a quick research about which ARIA role we should use in this case with this problem in mind.
So for me, the role section, implied by the section tag, sounds good, according to the role spec. Doing new research now, IMO, the role **banner** also make sense.

Does this make sense ? @filipewl @lucasfp13

Reply:
@igorbrasileiro by the example [given here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/banner_role) I don't think we can use `banner`, since this role is more related to the main page's header 😕